DangerSwap/DangerSwap/Repositories/UserRepository.cs

```diff
@@ -8,6 +8,7 @@ namespace DangerSwap.Repositories
 {
     public class UserRepository
     {
+        //Mindaugas
         private readonly UserManager<User> _userManager;
```

Comment:
Line 17 wrong parameter is passed as an argument of exception

Reply:
Line 36 has a wrong exception type. I'd recommend to change it to `ArgumentException` and write an explanatory message.

Reply:
One of the `GetEntity` methods could be removed as it is redundant.

Reply:
GetEntity return type should be `Task<User?>` as we return `FirstOrDefaultAsync` which means that if there's not desired record then it returns default value which is in our case `null`.

Reply:
Line 5 `using System.Linq;` is redundant.

Reply:
Line 29. Could use Guard Clause.